uqcsbot/__init__.py

```diff
@@ -26,27 +26,30 @@
 
 logger = logging.getLogger("uqcsbot")
 
-def is_active_bot(user_id):
+def get_bot_info(user_id):
```

Comment:
seeing as though this doesn't check for whether its a bot at all, I'd fix the description and rename it `get_user_info`